src/IW/Db/Repo.hs

```diff
@@ -18,6 +19,14 @@ getRepos = queryRaw [sql|
     FROM repos
 |]
 
+-- | Returns a list of repos filtered by category name
+getReposByCategory :: (WithDb env m) => Text -> m [Repo]
+getReposByCategory category = query [sql|
+    SELECT id, owner, name, descr, categories
+    FROM repos
+    WHERE ? = ANY (categories)
```

Comment:
This is okay for now. But in future, we will need to implement more complicated logic here. Because on the frontend, it should be possible to specify multiple categories and query all issues that has one of the specified categories. Same for labels. This will become relevant once we implement API endpoints and integration with the frontend. But you can create an issue about that so that we will keep track of such details.